ovirt_imageio/client/_options.py

```diff
@@ -45,13 +45,34 @@ def __repr__(self):
 log_level = Choices("log_level", ("debug", "info", "warning", "error"))
 
 
+class StoreFalseOrNoneAction(argparse.Action):
+    """
+    This action stores 'False' value, similar to 'store_false' but without
+    the implicit default 'True' value.
+    This action is to be used with optional arguments that specify
+    some sort of flag.
+    """
+
+    def __init__(self, option_strings, dest, required=False, help=None):
+        super().__init__(
+            option_strings=option_strings,
+            dest=dest,
+            nargs=0,
+            required=required,
+            help=help)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        setattr(namespace, self.dest, False)
```

Comment:
Is this called only when user specifies --insecure?

Reply:
Yes, I think so.